IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -73,6 +77,19 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
     MSIDRequestCompletionBlock completionBlockWrapper = ^(MSIDTokenResult * _Nullable result, NSError * _Nullable error)
     {
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Interactive flow finished. Result %@, error: %ld error domain: %@", _PII_NULLIFY(result), (long)error.code, error.domain);
+        if (!error)
+        {
+            /**
+             Throttling service: when an interactive token succeed, we update the last refresh time of the throttling service
+             */
+            if ([self.tokenRequestProvider isKindOfClass:MSIDDefaultTokenRequestProvider.class])
```

Comment:
This is a bit ugly - any way to avoid checking for class this way?

Reply:
Addressed